src/main/java/frc/robot/RobotContainer.java

```diff
@@ -130,12 +137,46 @@ protected Command getSimpleAuto(Pose2d startingPosition) {
     return trajectoryFactory.getPathFindToPathCommand(pathname, PathType.CHOREO);
   }
 
+  protected Command getOneNoteAuto() {
+    return trajectoryFactory.getPathFindToPoseCommand(AutoConstants.FIRST_NOTE_SHOOTING_POSITION)
+      .alongWith(new RotatePivotCommand(pivot, AutoConstants.FIRST_NOTE_SHOOTING_ANGLE.getRadians()).until(() -> pivot.getPosition().minus(AutoConstants.FIRST_NOTE_SHOOTING_ANGLE).getDegrees() < 2));
+      // .andThen(new SpinShooterCommand(shooter, ShooterConstants.lowerShooterSpeed, ShooterConstants.upperShooterSpeed));
+      // TODO: uncomment the above line when shooter is created
+  }
+
+  protected Command getTwoNoteAuto() {
+    return getOneNoteAuto().andThen(new PathPlannerAuto("2 Note Auto"));
+  }
+
+  protected Command getThreeNoteAuto() {
+    return getOneNoteAuto().andThen(new PathPlannerAuto("3 Note Auto"));
+  }
+
+  protected Command getFourNoteAuto() {
+    return getOneNoteAuto().andThen(new PathPlannerAuto("4 Note Auto"));
+  }
+
   public void smartDashSetup() {
-    autoChooser.addDefaultOption("Do Nothing", pose -> new WaitUntilCommand(() -> false));
+    autoChooser.addDefaultOption("Do Nothing", () -> new WaitUntilCommand(() -> false));
     autoChooser.addOption("Simple Auto", this::getSimpleAuto);
+    autoChooser.addOption("One-Note Auto", this::getOneNoteAuto);
+    autoChooser.addOption("Two-Note Auto", this::getTwoNoteAuto);
+    autoChooser.addOption("Three-Note Auto", this::getThreeNoteAuto);
+    autoChooser.addOption("Four-Note Auto", this::getFourNoteAuto);
 
     startingPositionChooser.addDefaultOption("Station 1", PathConstants.STATION_1);
     startingPositionChooser.addOption("Station 2", PathConstants.STATION_2);
     startingPositionChooser.addOption("Station 3", PathConstants.STATION_3);
   }
+
+  private void registerNamedCommands() {
+    /*
+    NamedCommands.registerCommand("intake", new SpinIntakeCommand(intake, IntakeConstants.INTAKE_SPEED));
+    NamedCommands.registerCommand("shoot", new SpinShooterCommand(shooter, ShooterConstants.lowerShooterSpeed, ShooterConstants.upperShooterSpeed));
+    */
+    NamedCommands.registerCommand("note1pivot", new RotatePivotCommand(pivot, AutoConstants.NOTE_1_SHOOTING_ANGLE.getRadians()).until(() -> pivot.getPosition().minus(AutoConstants.NOTE_1_SHOOTING_ANGLE).getDegrees() < 2));
+    NamedCommands.registerCommand("note2pivot", new RotatePivotCommand(pivot, AutoConstants.NOTE_2_SHOOTING_ANGLE.getRadians()).until(() -> pivot.getPosition().minus(AutoConstants.NOTE_2_SHOOTING_ANGLE).getDegrees() < 2));
```

Comment:
Perhaps use MathUtil.applyDeadband()? Additionally, please make a constant for the angle deadband for the arm.